### PR TITLE
O/add content field to nftinterface

### PR DESCRIPTION
**Pull request by olegakbarov**

### CODE REVIEW

File: src/types/NFTInterface.ts [hunk 1]
```diff
@@ -254,12 +254,20 @@ export type NFTObject = {
...
   };
   media?: {
-    thumbnail?: MediaObject;
+    source: MEDIA_SOURCES;
     image?: MediaObject;
     large?: MediaObject;
     poster?: MediaObject;
+    thumbnail?: MediaObject;
+    // backwards compatibility
     content?: MediaObject;
```
Comment (iainnash) [inline src/types/NFTInterface.ts:263]: We probably? safely can remove this content field?

Comment (olegakbarov) [reply src/types/NFTInterface.ts:263]: so idea is to retain backwards compatibility
i was thinking about going back to it when covering the whole thing with tests, wdyt?

**Comment by iainnash** [comment]
sounds good

File: src/types/NFTInterface.ts (hunk 1 above)
Comment (iainnash) [reply src/types/NFTInterface.ts:263]: sounds good

**Review by iainnash — APPROVED** [review]

**Pull request merged**